src/test/resources/pages/adminActivityLogPage.html

```diff
@@ -671,11 +552,11 @@ <h4 class="text-center">
 
 
 
-    <div class="alert alert-warning" id="statusMessage">
+    <div class="alert alert-info" id="statusMessage">
           Total Logs gone through in last search: {*}<br /><input type="hidden" value="false" id="ifShowAll" /><input type="hidden" value="false" id="ifShowTestData" /></div>
 
                 
-            {*}
+            <br />
```

Comment:
I think we should not remove the wildcard {*} here since in the staging server, there are many more logs and the "Olders Logs" will appear as in #4027. The wildcard here is to deal with that.